config/site_settings.yml

```diff
@@ -1442,6 +1442,8 @@ files:
   s3_cdn_url:
     default: ""
     regex: '^https?:\/\/.+[^\/]$'
+  s3_use_cdn_url_for_all_uploads:
```

Comment:
Is there a reason we went with a site setting? This kind of feels like a bug to me where uploads are not using the CDN when a CDN has been configured.

Reply:
> Is there a reason we went with a site setting? This kind of feels like a bug to me where uploads are not using the CDN when a CDN has been configured.

This is to have an escape hatch where the new behaviour is undesirable for some use cases (where CDN is expensive and you want those files to be served over S3 URL)